apps/src/lib/config/ethereum_bridge/params.rs

```diff
@@ -0,0 +1,106 @@
+//! Blockchain-level parameters for the configuration of the Ethereum bridge.
+use serde::{Deserialize, Serialize};
+
+/// Represents a configuration value for an Ethereum address e.g.
+/// 0x6B175474E89094C44Da98b954EedeAC495271d0F
+#[derive(Clone, Eq, PartialEq, Debug, Deserialize, Serialize)]
+#[repr(transparent)]
+pub struct Address(String);
+
+/// Represents a configuration value for the minimum number of
+/// confirmations an Ethereum event must reach before it can be acted on.
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Deserialize, Serialize)]
+#[repr(transparent)]
+pub struct MinimumConfirmations(u64);
```

Comment:
```suggestion
pub struct MinimumConfirmations(NonZeroU64);
```
maybe overkill, but this prevents you from configuring a value of 0 confirmations

Reply:
Definitely not overkill!! I had no idea there was this type in the std library, this works perfectly here